common/networking/packets.def

```diff
@@ -1083,6 +1083,7 @@ PACKET_UNIT_COMBAT_INFO = 65; sc, lsend
   HP defender_hp;
   BOOL make_att_veteran;
   BOOL make_def_veteran;
+  BOOL bombard;
```

Comment:
Adding this field breaks compatibility between old clients (that won't have it) and new servers (that will), or old servers and new clients. It may technically work because of how boolean fields are encoded into the packets, but I'm not sure.

There is a mechanism to add or remove fields without breaking network compat. This is done via the `add-cap` or `remove-cap` flags and a capability string.

Reply:
Ok, I'll ask you about this once we conclude our current discord discussion :+1:

Reply:
Removing these changes as they are no longer needed